CHANGES.rst

```diff
@@ -202,6 +202,10 @@ astropy.io.misc
 astropy.io.fits
 ^^^^^^^^^^^^^^^
 
+- Create magic methods ``__copy__`` and ``__deepcopy__`` on an ``HDUList``
+  class instance [#7218].
```

Comment:
Minor nitpick: The dot should be in front of the issue number `class instance. [#7218]`